tests/workflows/snvphylnfc.nf.test

```diff
@@ -66,4 +66,66 @@ nextflow_workflow {
 
     }
 
+    test("selecting reference by sample id") {
+
+        when {
+            params {
+                input = "https://raw.githubusercontent.com/phac-nml/snvphylnfc/409864a836dd4c2cd6191e7456881b973ca04748/assets/samplesheet.csv"
```

Comment:
Yeah, so the issue here is that whatever we point to needs to exist. The changes here add a new column, and the the version in main/dev does not have it. It doesn't seem right to make the change directly in the other branches, because nf-validation won't be set up for that new column. My thought was making a quick change once it's merged into dev.

Reply:
That makes perfect sense to me. Thanks.